server/api/src/rs_get_resource_info_for_operation.cpp

```diff
@@ -0,0 +1,85 @@
+#include "irods/get_resource_info_for_operation.h"
+#include "irods/rodsLog.h"
+#include "irods/rsGlobalExtern.hpp"
+#include "irods/rcGlobalExtern.h"
+#include "irods/getRemoteZoneResc.h"
+#include "irods/dataObjCreate.h"
+#include "irods/objMetaOpr.hpp"
+#include "irods/resource.hpp"
+#include "irods/collection.hpp"
+#include "irods/specColl.hpp"
+#include "irods/miscServerFunct.hpp"
+#include "irods/openCollection.h"
+#include "irods/readCollection.h"
+#include "irods/closeCollection.h"
+#include "irods/dataObjOpr.hpp"
+#include "irods/rsGetHostForGet.hpp"
+#include "irods/irods_resource_backport.hpp"
+#include "irods/irods_resource_redirect.hpp"
+#include "irods/irods_at_scope_exit.hpp"
```

Comment:
Please remove unused header includes.

Reply:
Unneeded as indicated by possible clangd/YouCompleteMe features?  Or codacy?

Reply:
Remove the obvious ones.

You can try commenting out each header to see if clangd reports issues.

Looking at that list, I'm guessing you don't need any of these.
- collection.hpp
- specColl.hpp?
- readCollection.h
- closeCollection.h
- rsGetHostForGet.hpp
- dataObjCreate.h
- rodsLog.h

Reply:
oh ... very cool , thanks